src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/ErrorHandling.java

```diff
@@ -25,7 +25,7 @@ public enum ErrorHandling {
 
   SKIP("Skip on error"),
 
-  STOP("Stop on error");
```

Comment:
Looks like this value is being used in some tests - https://github.com/data-integrations/salesforce/blob/cffa7e499e6ea80ff290640c062852c5d3f00942/src/test/java/io/cdap/plugin/salesforce/etl/BaseSalesforceBatchSinkETLTest.java#L191 . Can you fix those and add a unit test with the new type?